src/cadmium-playercore-shim.js

```diff
@@ -53,6 +51,52 @@ function get_profile_list() {
 		"BIF320"
 	];
 
+        if (my_config["use_avc.high"]) {
```

Comment:
It looks like you're mixing tabs and spaces here, please use tabs for indentation throughout